### PR TITLE
Handle React components in object properties.

### PR DESCRIPTION
**Pull request by ericsoderberghp**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -8,6 +8,24 @@ function isDefaultProp(defaultProps, key, value) {
   return defaultProps[key] === value;
 }
 
+function stringedObject (object, opts) {
+  var result = {};
+  Object.keys(object).map(function (key) {
+    var value = object[key];
+    if (isValidElement(value)) {
```
Comment (alansouzati) [inline src/index.js]: should we use `React.isValidElement` instead?

File: src/index.js [hunk 2]
```diff
@@ -39,6 +57,12 @@ function jsxToString(component, options) {
           } else if (React.isValidElement(value)) {
             value = jsxToString(value, opts);
           } else if (typeof value === 'object') {
+            value = stringify(stringedObject(value));
```
Comment (alansouzati) [inline src/index.js]: should we pass `opts` to `stringedObject` function?

File: src/index.js (hunk 1 above)
Comment (alansouzati) [inline src/index.js]: I believe the CI build is failing because of that.

Comment (ericsoderberghp) [inline src/index.js]: Yep. Fixed.

File: src/index.js (hunk 2 above)
Comment (ericsoderberghp) [inline src/index.js]: Yep. Fixed.

**Pull request merged**